### PR TITLE
Issue #2315: Link "Available updates" page to "Updates" page

### PR DESCRIPTION
**Pull request by laryn**

### CODE REVIEW

File: core/modules/update/theme/update.theme.inc [hunk 1]
```diff
@@ -105,7 +108,12 @@ function theme_update_report($variables) {
 
     $row = '<div class="version-status">';
     $status_label = theme('update_status_label', array('status' => $project['status']));
-    $row .= !empty($status_label) ? $status_label : check_plain($project['reason']);
+    if (empty($status_label)) $status_label = check_plain($project['reason']);
```
Comment (quicksketch) [inline core/modules/update/theme/update.theme.inc]: Coding standards: We always include squiggle brackets around IF statements. https://api.backdropcms.org/php-standards#control-structures

File: core/modules/update/theme/update.theme.inc [hunk 2]
```diff
@@ -78,10 +78,13 @@ function theme_update_report($variables) {
   }
 
   foreach ($data as $project) {
+    //Don't show a link to the "Update modules" page for core
+    $updatelink = ($project['name'] != 'backdrop');
```
Comment (quicksketch) [inline core/modules/update/theme/update.theme.inc]: Coding standards: We always separate words within a variable by an underscore, e.g. `$update_link`

File: core/modules/update/theme/update.theme.inc [hunk 3]
```diff
@@ -105,7 +108,12 @@ function theme_update_report($variables) {
 
     $row = '<div class="version-status">';
     $status_label = theme('update_status_label', array('status' => $project['status']));
-    $row .= !empty($status_label) ? $status_label : check_plain($project['reason']);
+    if (empty($status_label)) $status_label = check_plain($project['reason']);
+    if ($updatelink) {
+      $row .= l($status_label,'admin/'.$project['project_type'].'s/update',array('html' => TRUE));
```
Comment (quicksketch) [inline core/modules/update/theme/update.theme.inc]: Coding standards: Include a space on either side of periods when doing string concatenation. https://api.backdropcms.org/php-standards#string-concatenation

**Review by quicksketch — CHANGES_REQUESTED** [review]
Overall this looks great and is a good enhancement for UX. We just need to fix some coding standards here and I think we're good to go. :)

File: core/modules/update/theme/update.theme.inc (hunk 3 above)
Comment (laryn) [reply core/modules/update/theme/update.theme.inc]: Thanks -- modified those now and will keep in mind going forward! Thanks for the link to the documentation, I hadn't bookmarked that before.

**Comment by docwilmot** [comment]
Fixed with https://github.com/backdrop/backdrop/pull/1739

**Pull request closed without merging**